src/main/webapp/view/view.js

```diff
@@ -19,7 +19,10 @@ const PARAM_VIDEO_ID = 'video-id';
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
-initialize();
+getLecture().then((lecture) => {
```

Comment:
Do you think you can switch to only using await or only using .then()? I think it's bad practice to mix them
.
https://docs.google.com/document/d/1J1s2uxmLxeuS6dEj110weXz1ajhAJRhUk0C7fowrNrc/edit